src/model_repository_manager/model_lifecycle.h

```diff
@@ -313,6 +314,12 @@ class ModelLifeCycle {
       ModelInfo* model_info, const bool is_update,
       const std::function<void(Status)>& OnComplete,
       std::shared_ptr<LoadTracker> load_tracker);
+  // Calculate time to load model
+  void CalculateAndReportLoadTime(ModelInfo* loaded_model_info);
```

Comment:
#ifdef TRITON_ENABLE_METRICS

Reply:
I don't see any metric reporting functions in .h being guarded by `#ifdef TRITON_ENABLE_METRICS` any reason to change the convention?
See `IncrementPendingRequestCount` and `DecrementPendingRequestCount`

Reply:
Then that's fine.